zhaquirks/tuya/ts004f.py

```diff
@@ -412,3 +414,86 @@ class TuyaSmartRemote004F(EnchantedDevice):
             ENDPOINT_ID: 1,
         },
     }
+
+
+class TuyaSmartRemote004FSK(EnchantedDevice):
```

Comment:
Can you move this whole class above `TuyaSmartRemote004F`? (because that class only matches by model)

Reply:
Done